dev-client/src/screens/SiteNotesScreen/EditSiteNoteScreen.tsx

```diff
@@ -47,8 +47,14 @@ export const EditSiteNoteScreen = ({note}: Props) => {
   const dispatch = useDispatch();
   const [isSubmitting, setIsSubmitting] = useState(false);
 
+  const currentUser = useSelector(state => state.account.currentUser.data);
+  const currentUserIsAuthor = note.authorId === currentUser?.id;
+
   const handleUpdateNote = async ({content}: {content: string}) => {
-    if (!content.trim()) {
+    if (!currentUserIsAuthor) {
+      navigation.pop();
```

Comment:
Question: Do we expect this to ever happen? If so, when?

Reply:
Yes: the form's 'done' button needs to be pressable even if the user can't edit the form, because that's how the user exits the form. It seemed more reasonable to make this change to disallow edits than to e.g. add another callback to the already-complex form wrapper component.